keyform/templates/keyform/keydata_form.html

```diff
@@ -0,0 +1,55 @@
+{% load i18n %}
+
+<div class="keydata-form">
+    <table>
+        <tr>
+            <td>
+                {{ keydata_form.core_number.errors }}
+                {{ keydata_form.core_number.label_tag }}
+            </td>
+            <td>
+                {{ keydata_form.core_number }}
+            </td>
+            <td colspan="2">
+                {% if include_buttons %}
+                    <input type="button" value="{% trans 'Add Another Key' %}" id="generate_forms"/>
+                    <input type="button" value="{% trans 'Cancel' %}" id="delete-button"/>
```

Comment:
Either this id should have an underscore or the one above should have a `-`. I cannot remember which is correct though.